src/core/annotation.js

```diff
@@ -485,6 +485,7 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
     var dict = params.dict;
     var data = this.data;
 
+    data.annotationType = AnnotationType.WIDGET;
```

Comment:
This change is no longer necessary when you rebase the patch as it has been fixed already in #6757.